File: bench/ref/ackermann.py
import sys
sys.setrecursionlimit(1000000)
def ack(m,n): return n+1 if m==0 else (ack(m-1,1) if n==0 else ack(m-1,ack(m,n-1)))
print(ack(3,10))

File: bench/ref/ackermann.c
#include <stdio.h>
long ack(long m,long n){ return m==0 ? n+1 : (n==0 ? ack(m-1,1) : ack(m-1,ack(m,n-1))); }
int main(){ printf("%ld\n", ack(3,10)); return 0; }
